### PR TITLE
Tutorial transformación datos XML #960

### PR DESCRIPTION
**Pull request by jamotilla**
I´m trying to publish this tutorial, this is my first one so please check this and make shure the images are right.

### CODE REVIEW

File: es/lecciones/transformacion-datos-xml-xsl.md [hunk 1]
```diff
@@ -0,0 +1,525 @@
...
+- Pedro Carrillo
+- José Antonio Motilla
+difficulty: 1
+activity: transforming
+topics: [data-manipulation]
+review-ticket: https://github.com/programminghistorian/ph-submissions/issues/162
+original: transforming-xml-with-xsl
+redirect_from: /es/lessons/transforming-xml-with-xsl
+layout: lesson
+abstract:
```
Comment (mdlincoln) [inline es/lecciones/transformacion-datos-xml-xsl.md]: this needs to look like:

```
abstract: |
   "Con este...
```

Please note the `|` in line 28 as well as the tab space starting line 29

**Review by mdlincoln — DISMISSED** [review]

File: es/lecciones/transformacion-datos-xml-xsl.md (hunk 1 above)
Comment (jamotilla) [reply es/lecciones/transformacion-datos-xml-xsl.md]: Thanks @mdlincoln i think its done

**Review by mdlincoln — DISMISSED** [review]
OK good @jamotilla ! Now go check the Travis build log - it shows another error you need to fix: don't use full links to https://programminghistorian.org

**Comment by jamotilla** [comment]
Hi @mdlincoln, i think it´s done, i don´t know if i need to make other changes

**Review by mdlincoln — CHANGES_REQUESTED** [review]
The lesson page is looking good! One final addition is necessary: M. H. Beals needs to have an `es` bio in `_data/ph_authors.yml`

**Comment by jamotilla** [comment]
The bio is done @mdlincoln

**Review by mdlincoln — APPROVED** [review]

**Comment by mdlincoln** [comment]
It looks good to me, @jamotilla ! When you are ready, please merge this branch 😄

**Pull request merged**